staging/src/k8s.io/kubelet/pkg/cri/streaming/portforward/httpstream.go

```diff
@@ -242,8 +242,8 @@ Loop:
 // function for the given stream pair.
 func (h *httpStreamHandler) portForward(p *httpStreamPair) {
 	ctx := context.Background()
-	defer p.dataStream.Close()
-	defer p.errorStream.Close()
+	defer p.dataStream.Close()  //nolint: errcheck
```

Comment:
I wonder if we have the same problem as in here and we should Reset() instead of  Close()

https://github.com/kubernetes/kubernetes/pull/126718#discussion_r1782263866

Reply:
I see , discussed here https://github.com/kubernetes/kubernetes/pull/117493#discussion_r1543998504